From: Julian Andres Klode <jak@debian.org>
Date: Sat, 19 Jul 2014 13:43:53 +0200
Subject: Stub out va_copy in efistdarg.h

We are missing a definition for va_copy, causing va_copy to be undefined
in the produced library.

Reported-by: Julian Andres Klode <jak@debian.org>
Bug-Debian: https://bugs.debian.org/753682
---
 inc/efistdarg.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/inc/efistdarg.h b/inc/efistdarg.h
index 312eebc..d76b11a 100644
--- a/inc/efistdarg.h
+++ b/inc/efistdarg.h
@@ -25,4 +25,5 @@ typedef CHAR8 * va_list;
 #define va_start(ap,v)  ( ap = (va_list)&v + _INTSIZEOF(v) )
 #define va_arg(ap,t)    ( *(t *)((ap += _INTSIZEOF(t)) - _INTSIZEOF(t)) )
 #define va_end(ap)  ( ap = (va_list)0 )
+#define va_copy(d,s)  ((d) = (s))
 #endif
